config.go

```diff
@@ -553,3 +564,66 @@ func (b *Builder) Domainname() string {
 func (b *Builder) SetDomainname(name string) {
 	b.Docker.Config.Domainname = name
 }
+
+// Change Translates the Dockerfile syntax commands and then saves into the buildah struct
+func (b *Builder) Change(option string) error {
+	opt := strings.SplitN(option, " ", 2)
+	if len(opt) != 2 {
+		return fmt.Errorf("Change option %q must be separated by ' '", option)
+	}
+	switch strings.ToUpper(opt[0]) {
+	case "ANNOTATION":
```

Comment:
nit: It would be nice to have these case statements in alpha order unless you think some should have precedence over others for performance.